File: Modul-06/SOAL 2/PRAK602-2310817210009-MuhammadIbnuSina.py
print("Input: ")
def BERAPA_JUMLAH_ZETSU(JUMLAH_RUANGAN, JUMLAH_ZETSU):
    hasil = []
    for i in range(JUMLAH_RUANGAN):
        hasil.append(JUMLAH_ZETSU[i] * (i + 1))
    return hasil

JUMLAH_RUANGAN = int(input())
JUMLAH_ZETSU = list(map(int, input().split()))
HASIL_RUANGAN = BERAPA_JUMLAH_ZETSU(JUMLAH_RUANGAN, JUMLAH_ZETSU) 

print("\nOutput: ")
print(" ".join(map(str, HASIL_RUANGAN)))

File: Modul-06/SOAL 2/PRAK602-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int TOTALRUANGAN, i;
    printf("Input: \n");
    scanf("%d", &TOTALRUANGAN);

    int ZETSUPUTIH[TOTALRUANGAN];
    for (i = 0; i < TOTALRUANGAN; i++) {
        scanf("%d", &ZETSUPUTIH[i]);
    }
    for (i = 0; i < TOTALRUANGAN; i++) {
        ZETSUPUTIH[i] = ZETSUPUTIH[i] * (i + 1);
    }

    printf("\nOutput: \n");
    for (i = 0; i < TOTALRUANGAN; i++) {
        printf("%d ", ZETSUPUTIH[i]);
    }

    return 0;
}